source_docs/installation/installation.md

```diff
@@ -4,13 +4,13 @@ Installing Bolt
 This page explains the various ways of installing Bolt. You can either use the
 command-line or your FTP-client to install it.
 
-There are several methods to install Bolt, but we recommend the "easiest way"
+There are several methods to install Bolt, but we recommend **the "easiest way"**
 [using the server command-line](install-command-line).
```

Comment:
...which means that you misread the page as well, it's **4** options altogether (CLI, FTP, Developers, Contributor).

Maybe we should then  put all options in the bullets or an ordered list to put them all in the same list.